components/PostContent/PostContent.tsx

```diff
@@ -1,3 +1,4 @@
+import config from '../../next.config';
```

Comment:
I added the `config` in lowercase because is the pattern on all files importing `next.config.js` file